src/vs/workbench/contrib/notebook/browser/diff/cellComponents.ts

```diff
@@ -750,30 +801,21 @@ export class DeletedCell extends AbstractCellRenderer {
 		super(notebookEditor, cell, templateData, 'left', instantiationService, modeService, modelService, contextMenuService, keybindingService, notificationService, menuService, contextKeyService);
 	}
 
-	initData(): void {
-	}
-
 	styleContainer(container: HTMLElement) {
 		container.classList.add('removed');
 	}
 
-	buildSourceEditor(sourceContainer: HTMLElement): void {
+	updateSourceEditor(): void {
 		const originalCell = this.cell.original!;
 		const lineCount = originalCell.textBuffer.getLineCount();
 		const lineHeight = this.notebookEditor.getLayoutInfo().fontInfo.lineHeight || 17;
 		const editorHeight = lineCount * lineHeight + getEditorTopPadding() + EDITOR_BOTTOM_PADDING;
 
-		const editorContainer = DOM.append(sourceContainer, DOM.$('.editor-container'));
-
-		this._editor = this.instantiationService.createInstance(CodeEditorWidget, editorContainer, {
-			...fixedEditorOptions,
-			dimension: {
-				width: (this.notebookEditor.getLayoutInfo().width - 2 * DIFF_CELL_MARGIN) / 2 - 18,
-				height: editorHeight
-			},
-			overflowWidgetsDomNode: this.notebookEditor.getOverflowContainerDomNode()
-		}, {});
-		this._register(this._editor);
+		this._editor = this.templateData.sourceEditor;
+		this._editor.layout({
+			width: (this.notebookEditor.getLayoutInfo().width - 2 * DIFF_CELL_MARGIN) / 2 - 18,
```

Comment:
The 18 is a magic number.  A named constant would help understand what this is for.  That said, as if is a copy-paste then we can just skip adding the constant if you do not know.